force-app/main/default/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -118,6 +138,24 @@ export default class rd2EntryForm extends LightningElement {
 
     @track error = {};
 
+    cssComponent = 'slds-section slds-is-open';
+    cssModalContent = 'slds-modal__content slds-p-top_none slds-p-horizontal_medium slds-p-bottom_medium';
+    ariaHidden = false;
+
+    @api isBankPaymentAllowed;
+    @track
+    paymentMethodOptions = [{ label: CreditCardPaymentMethod, value: CREDIT_CARD }];
+
+    determineACHpaymentMethodAndAddAsOption() {
```

Comment:
Can we add a jest test to validate the logic we added here for determining if ACH is a valid payment option?

Reply:
Ok, I'll update the jest test to check this feature.